### ISSUE TITLE
[Accessibility] DetailsList's Column header text is not being read with its state of sorting, i.e., "ascending" or "descending"

### ISSUE THREAD

**Issue by kmrrakesh** [OfficeDev/office-ui-fabric-react#4870]
**Bug Report**
**Describe the issue :**
Screen-Reader is not reading DetailsList column header text with "ascending" or "descending".

I have used the DetailsList and did accessibility testing on it using narrator with edge browser, at header columns of grid, with CAPS LOCK + LEFT/RIGHT arrow key.

**Actual behavior:**
Column header text is not being read with its state of sorting, i.e., "ascending" or "descending"

**Expected behavior:**
Column header text should be read with its state of sorting, i.e., "ascending" or "descending"

**Comment by JasonGore** [comment]
If you know of a grid example that actually reads sort mode with Narrator in Edge, please let me know. I could not find one.

DetailsList output seems to be properly formatted using the defined roles and `aria-sort` property according to the ARIA specification: https://www.w3.org/TR/wai-aria-1.1/#aria-sort

I have done some investigation and couldn't find an example making use `aria-sort`, such as https://dequeuniversity.com/library/aria/tables/sf-sortable-grid, that actually verbalize in any of the following browsers using Narrator: Chrome, Edge and Firefox.

I have further found that "ascending" and "descending" are in fact verbalized properly for both DetailsList and the example above when using JAWS with Chrome. As such, I think this is Narrator and/or browser-specific behavior preventing `aria-sort` from being verbalized. I will follow up with those teams regarding this issue.

**Comment by arunarokk** [comment]
@JasonGore , 
 https://developer.microsoft.com/en-us/fabric#/components/detailslist   if you verify first detailslist in this link with narrator on, instead of using tab if you use CapsLock + right arrow , the focus goes to  div of the Name column header where the aria-sort attribute is defined, narrator announcing the sort order ascending / descending. I guess we need to correct the tab index here to place the focus on right element.

**Comment by JasonGore** [comment]
That behavior is the same as for the ARIA grid reference I posted above as well as this one here: https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/grid/dataGrids.html. Even following ARIA spec, the behavior is as you mentioned.

I've gotten confirmation that this is an issue from the Narrator team in that `aria-sort` does not map to a UIA property and, as a result, Narrator will not verbalize it. However, I've asked for clarification for the detail you mentioned in which CAPS + left/right does verbalize it, which seems to indicate it supports it in some fashion.

**Comment by JasonGore** [comment]
I have received acknowledgement from the Narrator team that this is a Narrator bug and have entered a bug for it, so I will close this issue. (Narrator is only promoting sorting order when the element is highlighted, as arunarokk described, which is a Narrator bug.)

**Comment by arunarokk** [comment]
@JasonGore , Thanks for the update, could you please let us know the Narrator bug reference?

**Comment by JasonGore** [comment]
@arunarokk Who is us? I cannot tell if you work at MS. This is a public forum and the Narrator issue is internal.
